lib/generator.js

```diff
@@ -580,6 +583,21 @@ class Generator {
     await writeFile(targetFile, parsedContent, { encoding: 'utf8', mode: stats.mode });
   }
 
+  /**
+   * It may rename the source file name in cases where special names are not supported, like .gitignore or .npmignore.
```

Comment:
would be great to have clearly mentioned here what we have in pr description, that it is here because of npm